pallets/teerex/src/lib.rs

```diff
@@ -102,6 +102,15 @@ pub mod pallet {
 	#[pallet::getter(fn allow_sgx_debug_mode)]
 	pub type AllowSGXDebugMode<T: Config> = StorageValue<_, bool, ValueQuery>;
 
+	#[pallet::type_value]
+	pub fn HeartbeatTimeoutDefault() -> u64 {
+		172_800_000
+	} // 48h
+	#[pallet::storage]
+	#[pallet::getter(fn heartbeat_timeout_storage)]
+	pub type HeartbeatTimeoutStorage<T: Config> =
+		StorageValue<_, u64, ValueQuery, HeartbeatTimeoutDefault>;
```

Comment:
Can we use `Moment` as type?

Reply:
As mentioned in the stand-up, `BlockNumber` is a better option

Reply:
How do you think we open another issue to take care of "timestamp" -> "blocknumber" ?